content/source/docs/extend/testing/acceptance-tests/teststep.html.md

```diff
@@ -33,7 +33,7 @@ Reference API](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/help
 
 ## Steps
 
-`Steps` is slice property of
+`Steps` is a slice property of
```

Comment:
I _think_ this is correct: grammatically "x is y" is wrong, but I'm not sure if "a slice property" is considered a different thing to "a property of type slice".